cmake/Corrosion.cmake

```diff
@@ -1092,6 +1092,25 @@ function(corrosion_link_libraries target_name)
     endforeach()
 endfunction()
 
+#[=======================================================================[.md:
+ANCHOR: corrosion-install
+```cmake
+corrosion_install(TARGETS <target1> ... <targetN> [EXPORT <export-name>]
+                  [[ARCHIVE|LIBRARY|RUNTIME|PRIVATE_HEADER|PUBLIC_HEADER]
```

Comment:
When documenting `corrosion_install` we should take care to only document what is really working. For example, `PRIVATE_HEADER` doesn't seem to be used in the code, and also doesn't make sense, since Rust code wouldn't have private header files.
I would not document any header install options yet (and I think it would make sense to do headers in a separate PR, since some discussion is probably required)

Reply:
Will fix.

Do you have an opinion on calling it `PUBLIC_HEADER` vs something like `PUBLIC_INCLUDES`?

It seems like [`PUBLIC_HEADER`](https://cmake.org/cmake/help/latest/prop_tgt/PUBLIC_HEADER.html) is a concept that has something to do with something Apple specific, although I have also seen people using `PUBLIC_HEADER` to install normal public header files.

Reply:
Fixed, but still interested in naming for headers. Is an easy fix so it could also just be discussed in the header PR.